docs/docs/monitoring.md

```diff
@@ -20,10 +20,17 @@ for all OpenTelemetry-related configurables.
 
 ## Prometheus
 
-OPA exposes an HTTP endpoint that can be used to collect performance metrics
+OPA exposes an HTTP endpoint at `/metrics` that can be used to collect performance metrics
 for all API calls. The Prometheus endpoint is enabled by default when you run
 OPA as a server.
 
+OPA provides two ways to access performance metrics:
+
+1. **System-wide metrics** via the `/metrics` Prometheus endpoint - Instance-level metrics across all OPA operations
+2. **Per-query metrics** via API responses with `?metrics=true` - Metrics for individual query executions
```

Comment:
I would put this in an [admonition ](https://docusaurus.io/docs/markdown-features/admonitions) instead since it's related but not 100% on topic for the prometheus section, this section is just about `/metrics` but ?metrics=true is important further reading.